languages/tgmpa-cs_CZ.po

```diff
@@ -0,0 +1,499 @@
+# Translation of TGM-Plugin-Activation in Czech.
+# This file is distributed under the same license as the TGMPA package.
+msgid ""
+msgstr ""
+"Project-Id-Version: TGM Plugin Activation\n"
```

Comment:
That is inconsistent with how GlotPress serves translations and fills in the `Project-Id-Version` header. Which should be leading in this case ?

Side-note - looks like I just discovered a bug in GlotPress - was checking how it served the headers and for themes it uses just the theme name (no version nr), for plugins it says 'stable (latest release)' without the plugin name - also for the header at the very top of the files.

Off to find out where I can fix/report this.